src/components/homePage/heroSection.jsx

```diff
@@ -0,0 +1,16 @@
+import heroContent from "../../data/landing-page-content.json";
+
+export function HeroSection() {
```

Comment:
@H-chai, I’d like to call attention to an area for improvement.

The current approach to handling data in React.js isn’t the most effective. As it stands, the component won't be reactive to changes in the data, which can lead to issues with updates or state management.

A better approach would be to pass the data as props. Let the parent component handle and send the title and subtitle to the Hero section as props. This ensures the component remains reactive and easier to maintain.

This is just a friendly reminder for future reference. There is no cause for concern.

Thank you for your contributions; we truly appreciate your efforts. Cheers!